de.dlr.sc.virsat.model.extension.tests.test/src/de/dlr/sc/virsat/model/dvlm/mat/MatImporterTest.java

```diff
@@ -0,0 +1,343 @@
+		assertTrue("same number of elements", caSei.size() == caSei2.size());
+		assertEquals("same testString", tc1.getTestString(), tc.getTestString());
+		assertEquals("same testBool", tc1.getTestBool(), tc.getTestBool());
+		assertEquals(tc1.getTestFloat(), tc.getTestFloat(), 0);
+		assertEquals("same testResource", tc1.getTestResource(), tc.getTestResource());
+		assertEquals("same testEnum", tc1.getTestEnum(), tc.getTestEnum());
+		assertEquals("same testInt", tc1.getTestInt(), tc.getTestInt());
+	}
+
+	@Test
+	public void testImportOfValuesRef() throws IOException {
+		//empty and import empty
+		TestCategoryReference tc = new TestCategoryReference(testConcept);
+		tsei.add(editingDomain, tc);
+		mat = exporter.exportSei(sei);
+		MatFile mat1 = exporter.exportSei(sei);
+		TestStructuralElement tsei2 = new TestStructuralElement(testConcept);
+		StructuralElementInstance sei2 = tsei2.getStructuralElementInstance();
+		TestCategoryReference tc1 = new TestCategoryReference(testConcept);
+		tsei2.add(editingDomain, tc1);
+		sei2.setName("testsei");
+		sei2.setUuid(sei.getUuid());
+		importer.importSei(sei2, mat);
+		assertEquals("same EReference from empty to empty", tc1.getTestRefCategory(), tc.getTestRefCategory()); //empty to empty
+
+		//values and import empty
+		TestCategoryAllProperty tc2 = new TestCategoryAllProperty(testConcept);
+		tc1.setTestRefCategory(editingDomain, tc2);
+		importer.importSei(sei2, mat1);
+		assertEquals("same EReference from value to empty", tc1.getTestRefCategory(), tc.getTestRefCategory());
+
+		//empty and import values
+		tc1.setTestRefCategory(editingDomain, tc2);
+		editingDomain.saveAll();
+		mat = exporter.exportSei(sei2);
+		importer.importSei(sei, mat);
+		assertEquals("same Reference from empty to value", tc1.getTestRefCategory(), tc.getTestRefCategory());
+
+		//values and import values
+		tc.setTestRefCategory(editingDomain, tc2);
+		tc1.setTestRefCategory(editingDomain, tc2);
+		editingDomain.saveAll();
+		mat = exporter.exportSei(sei2);
+		importer.importSei(sei, mat);
+		assertEquals("same Reference from value to value", tc1.getTestRefCategory(), tc.getTestRefCategory());
+	}
+
+	@Test
+	public void testImportOfValuesERef() throws IOException {
+		ExternalTestType testERef = de.dlr.sc.virsat.model.external.tests.TestsFactory.eINSTANCE.createExternalTestType();
+		
+		//empty and import empty
+		EReferenceTest tc = new EReferenceTest(testConcept);
+		tsei.add(editingDomain, tc);
+		mat = exporter.exportSei(sei);
+		MatFile mat1 = exporter.exportSei(sei);
+		TestStructuralElement tsei2 = new TestStructuralElement(testConcept);
+		StructuralElementInstance sei2 = tsei2.getStructuralElementInstance();
+		EReferenceTest tc1 = new EReferenceTest(testConcept);
+		tsei2.add(editingDomain, tc1);
+		sei2.setName("testsei");
+		sei2.setUuid(sei.getUuid());
+		importer.importSei(sei2, mat);
+		assertEquals("same Reference from empty to empty", tc1.getEReferenceTest(), tc.getEReferenceTest());

```

Comment:
I think it makes more sense to test the value of an EReference here... rather than comparing the property...